### ISSUE TITLE
Is EEP 3.0 respected?

### ISSUE THREAD

**Issue by Jey-Cee** [enocean-js/enocean-js#171]
While i am looking around i found the specs for EEP in version 3.0. 
There are a important change to RORG, FUNC and TYPE identification, they now all have a length of 8-bit. In <3.0 it was 8, 6 and 7-bit.

Did you saw this already @Holger-Will?

**Comment by Holger-Will** [comment]
no, haven't seen this update at yet, thanks for the hint... as I read the spec this will only be the case for FUNC > 3f or TYPE > 7f, right?

**Comment by Jey-Cee** [comment]
I have not read the details.

**Comment by Holger-Will** [comment]
> From EEP 3.0 we define all fields with 8 bits. This has impact for 4BS Teach-In, if FUNC > 3F, or
> TYPE >7F, and in Remote Management, but will be explained there.

that is the relevant part...
